Bottom: 83416e89ce314fa670b1638f6bde064da0f1d1b7
Top:    643b83285b47dc0791ef1918a80396e3184aabdb
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 21:14:33 -0700

fixup-i-dont-know-where


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index fe6eadbe229..599de5db69f 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -552,7 +552,6 @@ class FileHandleTestCase(lldbtest.TestBase):
 
     @add_test_categories(['pyapi'])
     @skipIf(py_version=['<', (3,)])
-    @skipIf(True) # fixme multiple problems with this
     def test_string_out(self):
         f = io.StringIO()
         status = self.debugger.SetOutputFile(f)
